software/sentryshot.yml

```
@@ -0,0 +1,12 @@
+name: "SentryShot"
+website_url: "https://github.com/SentryShot/sentryshot"
+source_code_url: "https://codeberg.org/SentryShot/sentryshot"
+description: "Video Management System"
+licenses:
+  - GPL-2.0
+platforms:
+  - Rust
+  - Docker
```

Just a quick change in order to match the docs

```suggestion
platforms:
  - Docker
  - Rust
```